libs/luci-lib-nixio/Makefile

```diff
@@ -10,6 +10,38 @@ LUCI_TITLE:=NIXIO POSIX library
 LUCI_DEPENDS:=+liblua
 
 PKG_LICENSE:=Apache-2.0
+PKG_CONFIG_DEPENDS:= \
+	CONFIG_LUCI_NIXIO_NOTLS \
+	CONFIG_LUCI_NIXIO_WOLFSSL \
+	CONFIG_LUCI_NIXIO_OPENSSL
+	
+define Package/luci-lib-nixio/config
+	choice
+		prompt "TLS Provider"
+		default LUCI_NIXIO_NOTLS
+
+		config LUCI_NIXIO_NOTLS
+			bool "Disabled"
+
+		config LUCI_NIXIO_WOLFSSL
+			bool "wolfSSL"
+
+		config LUCI_NIXIO_OPENSSL
+			bool "OpenSSL"
+	endchoice
+endef
+
+export NIXIO_TLS :=
+
+ifneq ($(CONFIG_LUCI_NIXIO_WOLFSSL),)
+	NIXIO_TLS:=wolfssl
+	LUCI_DEPENDS += +libwolfssl
```

Comment:
```suggestion
```
The package dependencies are written to `tmp/.packagedeps` before config is read, so this won't work.  Use  conditional dependencies above.